pkg/server/transform.go

```diff
@@ -173,42 +173,42 @@ func messageSliceFromProto(m []*translatev1.Message) ([]model.Message, error) {
 
 // ----------------------Messages----------------------
 
-// messagesToProto converts *model.Messages to *translatev1.Messages.
-func messagesToProto(m *model.Messages) *translatev1.Messages {
-	if m == nil {
+// translationToProto converts *model.Translation to *translatev1.Translation.
+func translationToProto(t *model.Translation) *translatev1.Translation {
+	if t == nil {
 		return nil
 	}
 
-	return &translatev1.Messages{
-		Language: languageToProto(m.Language),
-		Original: m.Original,
-		Messages: messageSliceToProto(m.Messages),
+	return &translatev1.Translation{
+		Language: languageToProto(t.Language),
+		Original: t.Original,
+		Messages: messageSliceToProto(t.Messages),
```

Comment:
As we do not have name stutter anymore this function name also should be changed
```suggestion
		Messages: messagesToProto(t.Messages),
```